src/main/webapp/css/srp_style.css

```diff
@@ -42,28 +42,64 @@ h1 {
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  max-height: 260px;
+  overflow: scroll;
+  font: 16px Roboto, Arial, sans-serif;
+  margin-top: 20px;
 }
 
-.card .card-title {
-  font: 300 20px Roboto, Arial, sans-serif;
+.card::-webkit-scrollbar {
+  width: 8px;
+  height: 8px;
+  background-color: #FAFAFA;
 }
 
-.card .card-description {
-  font: bold 16px Roboto, Arial, sans-serif;
-  margin-top: 20px;
+.card::-webkit-scrollbar-thumb {
+  background: lightgrey;
+  border-radius: 10px;
+}
+
+.card::-webkit-scrollbar-thumb:hover {
+  background: grey;
+}
+
+.card .card-title {
+  font: 300 20px Roboto, Arial, sans-serif;
 }
 
 .card code {
-  margin-top: 10px;
-  margin-bottom: 10px;
+  margin: 10px 5px 5px 10px;
```

Comment:
I think you meant for this to be `10px 5px 10px 5px` right now the 10px is being set on top and left.